.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Continuous Integration
+
+on: [ push ]
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
```

Comment:
Is there a way to separately run this on macos-latest as well? I've accidentally broken the build on macOS before (see #40) because I develop on Linux and it wasn't running some darwin-only tests.

Reply:
Changed to:

```
jobs:
  test:
    strategy:
      matrix:
        os: [ 'macos-latest', 'ubuntu-16.04' ]
        go: [ '1.13' ]

    runs-on: ${{ matrix.os }}
```